src/server/game/Server/GameClient.h

```diff
@@ -38,6 +38,8 @@ class TC_GAME_API GameClient
         void SetActiveMover(Unit* activeMover) { _activeMover = activeMover; }
 
         Player* GetBasePlayer() const { return _sessionToServer->GetPlayer(); }
+
+        void SendDirectMessage(WorldPacket const* data) const { GetBasePlayer()->SendDirectMessage(data); }
```

Comment:
this line breaks no-pch and I would say having it in .cpp is better than including Player.h in this header (no idea why people are obsessed of moving everything to headers as 1 line functions that are a pain to edit and debug with breakpoints)

Reply:
Done